platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/PKIHandlerTest.java

```diff
@@ -160,12 +186,7 @@ public void testCertPassesCRLCheckWhenNotListedInCRL()
         X509Certificate[] certs = new X509Certificate[1];
         certs[0] = cert;
 
-        BaseAuthenticationToken token = new BaseAuthenticationToken(new Object(), "test",
-                new Object());
-        HandlerResult result = new HandlerResult();
-
-        result = handler.crlChecker.check(token, certs, result);
-        assertThat(result.getStatus(), equalTo(HandlerResult.Status.COMPLETED));
+        assertThat(handler.crlChecker.passesCrlCheck(certs), equalTo(true));
```

Comment:
We should have a separate `CrlCheckerTest` class and have these assertions in there instead. This test class should really just focus on testing the `PKIHandler`'s behavior.